tests/automated_validation/data_transformation/test_measures.py

```diff
@@ -0,0 +1,138 @@
+import pandas as pd
+
+from vivarium_testing_utils.automated_validation.data_transformation.measures import (
+    Incidence,
+    Prevalence,
+    SIRemission,
+)
+
+
+def test_incidence(
+    transition_count_data: pd.DataFrame, person_time_data: pd.DataFrame
+) -> None:
+    """Test the Incidence measure."""
+    cause = "disease"
+    measure = Incidence(cause)
+    assert measure.measure_key == f"cause.{cause}.incidence_rate"
+    assert measure.sim_datasets == {
+        "numerator_data": f"transition_count_{cause}",
+        "denominator_data": f"person_time_{cause}",
+    }
+    assert measure.artifact_datasets == {"artifact_data": measure.measure_key}
+
+    ratio_data = measure.get_ratio_data_from_sim(
+        numerator_data=transition_count_data,
+        denominator_data=person_time_data,
+    )
+    expected_ratio_data = pd.DataFrame(
+        {
+            "susceptible_to_disease_to_disease_transition_count": [3, 5],
+            "susceptible_to_disease_person_time": [17, 29],
+        },
+        index=pd.Index(
+            ["A", "B"],
+            name="stratify_column",
+        ),
+    )
+    assert ratio_data.equals(expected_ratio_data)
+
+    measure_data_from_ratio = measure.get_measure_data_from_ratio(ratio_data=ratio_data)
+
+    measure_data = measure.get_measure_data_from_sim(
+        numerator_data=transition_count_data, denominator_data=person_time_data
+    )
+    expected_measure_data = pd.Series(
+        [3 / 17, 5 / 29],
+        index=pd.Index(
+            ["A", "B"],
+            name="stratify_column",
+        ),
+    )
+    assert measure_data.equals(expected_measure_data)
+    assert measure_data_from_ratio.equals(expected_measure_data)
+
+
+def test_prevalence(person_time_data: pd.DataFrame) -> None:
+    """Test the Prevalence measure."""
+    cause = "disease"
+    measure = Prevalence(cause)
+    assert measure.measure_key == f"cause.{cause}.prevalence"
+    assert measure.sim_datasets == {
+        "numerator_data": f"person_time_{cause}",
+        "denominator_data": f"person_time_{cause}",
+    }
+    assert measure.artifact_datasets == {"artifact_data": measure.measure_key}
+
+    ratio_data = measure.get_ratio_data_from_sim(
+        numerator_data=person_time_data,
```

Comment:
this is confusing to me how the numerator data and denominator data can be the same dataset. What's actually going on here?